django/db/backends/base/schema.py

```diff
@@ -942,6 +942,9 @@ def _constraint_names(self, model, column_names=None, unique=None,
         Returns all constraint names matching the columns and conditions
         """
         column_names = list(column_names) if column_names else None
+        if column_names and self.connection.features.uppercases_column_names:
+            # Oracle's get_constraints() returns all columns as lower case.
```

Comment:
No, as lower case.

See the use of `LOWER()` in the query [here](https://github.com/django/django/blob/8db889eaf7dce0cb715b075be32047c1b1b316da/django/db/backends/oracle/introspection.py#L158).